### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by arielkru**
Missing:
* updated links to marketplace throughout MD + link to apache repo + right links in badges
* GIF of plugin working

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +45,57 @@ To access logs directory, go to Help -> Show log in IDE.
...
+* Checkov will run automatically everytime an IaC is opened or saved.
+* Scan results should now appear in the checkov tool window in the bottom of your IDE.
+* Scan results will appear on the left side as a tree of File Names -> Resources -> Violated checks.
+* Click a check to see its details. Details including  violating policy and a link to step-by-step fix guidelines.
+* In most cases, the Details will include a fix option. This will either add, remove or replace an unwanted configuration, based on the Checkov fix dictionaries.
+* You can skip checks by adding an inline skip annotation ```checkov:skip=<check_id>:<suppression_comment>```. For more details see the [docs](https://github.com/bridgecrewio/checkov/blob/master/docs/2.Concepts/Suppressions.md).
+
+### Troubleshooting logs
+
+To access checkov-intellij logs directory, go to `Help` and select `Show Log in Finder`.
```
Comment (orhovy) [inline README.md:74]:
```suggestion
To access checkov-intellij logs directory, go to `Help` and select `Show Log in Finder` or `Show Log in Explorer`.
```

File: README.md [hunk 2]
```diff
@@ -1,55 +1,38 @@
...
 
 
-###Configuration
-Sign up to a Bridgecrew Community account  [here](https://https://www.bridgecrew.cloud/).. If you already have an account, sign in and go to the next step.
+#TODO PLUGIN GIF DEMO
 
-From Integrations, select API Token and copy the API key.
 
-In Jetbrains, enter your API Token in the Checkov settings page under tools.
+The extension is currently available for download directly from the [IntelliJ Plugin Marketplace](link-to-plugin.com) and its source code is available in an [Apache 2.0 licensed repository](link-to-license.com).
```
Comment (orhovy) [inline README.md:19]:
```suggestion
The pluginm is currently available for download directly from the [IntelliJ Plugin Marketplace](link-to-plugin.com) and its source code is available in an [Apache 2.0 licensed repository](link-to-license.com).
```

File: README.md [hunk 3]
```diff
@@ -62,7 +45,57 @@ To access logs directory, go to Help -> Show log in IDE.
   <kbd>Settings/Preferences</kbd> > <kbd>Plugins</kbd> > <kbd>⚙️</kbd> > <kbd>Install plugin from disk...</kbd>
 
 
+### Dependencies
+
+* [Python](https://www.python.org/downloads/) >= 3.7 or [Pipenv](https://docs.pipenv.org/) or [Docker](https://www.docker.com/products/docker-desktop) daemon running
+
+The Checkov extension will invoke the latest version of ```Checkov```.
```
Comment (orhovy) [inline README.md:52]:
```suggestion
The Checkov plugin will invoke the latest version of ```Checkov```.
```

File: README.md [hunk 4]
```diff
@@ -62,7 +45,57 @@ To access logs directory, go to Help -> Show log in IDE.
...
+
+Contribution is welcomed!
+
+Start by reviewing the [contribution guidelines](https://github.com/bridgecrewio/checkov/blob/master/CONTRIBUTING.md). After that, take a look at a [good first issue](https://github.com/bridgecrewio/checkov/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue").
+
+Looking to contribute new checks? Learn how to write a new check (AKA policy) [here](https://github.com/bridgecrewio/checkov/blob/master/docs/5.Contribution/New-Check.md).
+
+## Disclaimer
+
+To use this checkov-intellij extension, you will need to create a free account at bridgecrew.cloud using your e-mail, the plugin uses Bridgecrew.cloud's fixes API to analyse and produce code fixes, and enrich the results provided into IntelliJ. Please notice bridgecrew [privacy policy](https://bridgecrew.io/privacy-policy/?utm_source=github&utm_medium=organic_oss&utm_campaign=checkov-vscode) for more details on collected data when using bridgecrew application.
```
Comment (orhovy) [inline README.md:86]:
```suggestion
To use this checkov-jetbrains extension, you will need to create a free account at bridgecrew.cloud using your e-mail, the plugin uses Bridgecrew.cloud's fixes API to analyse and produce code fixes, and enrich the results provided into IntelliJ. Please notice bridgecrew [privacy policy](https://bridgecrew.io/privacy-policy/?utm_source=github&utm_medium=organic_oss&utm_campaign=checkov-vscode) for more details on collected data when using bridgecrew application.
```

File: README.md [hunk 5]
```diff
@@ -62,7 +45,57 @@ To access logs directory, go to Help -> Show log in IDE.
...
+[Bridgecrew](https://bridgecrew.io/?utm_source=github&utm_medium=organic_oss&utm_campaign=checkov-vscode) builds and maintains Checkov to make policy-as-code simple and accessible.
+
+Start with our [Documentation](https://bridgecrewio.github.io/checkov/) for quick tutorials and examples.
+
+If you need direct support you can contact us at [info@bridgecrew.io](mailto:info@bridgecrew.io).
+
+
 ---
-Plugin based on the [IntelliJ Platform Plugin Template][template].
+The plugin is based on the [IntelliJ Platform Plugin Template][template].
```
Comment (orhovy) [inline README.md:99]:
```suggestion
The plugin is based on the [Jetbrains Platform Plugin Template][template].
```

File: README.md (hunk 1 above)
Comment (arielkru) [reply README.md:74]: Perfect, the second is for windows right? so it makes sense if I add (for Mac) and (for Windows) after each @orhovy

Comment (orhovy) [reply README.md:74]: yes

**Pull request merged**